src/main/java/com/mparticle/kits/AdobeKit.kt

```diff
@@ -90,20 +91,24 @@ open class AdobeKit: AdobeKitBase(), KitIntegration.EventListener {
     }
 
     private fun sessionStart(mediaEvent: MediaEvent) {
+        val sessionId = mediaEvent.sessionId ?: return
         val mediaInfo = mediaEvent.mediaContent.getMediaObject()
-        mediaTracker?.trackSessionStart(mediaInfo, mediaEvent.customAttributes?.toAdobeAttributes())
+        mediaTrackers[sessionId] = Media.createTracker()
+        mediaTrackers[sessionId]?.trackSessionStart(mediaInfo, mediaEvent.customAttributes?.toAdobeAttributes())
     }
 
-    private fun sessionEnd() {
-        mediaTracker?.trackSessionEnd()
+    private fun sessionEnd(mediaEvent: MediaEvent) {
+        val sessionId = mediaEvent.sessionId ?: return
+        mediaTrackers[sessionId]?.trackSessionEnd()
+        mediaTrackers[sessionId] = null
```

Comment:
When you change the map to `MutableMap<String, MediaTracker>` this will no longer be:

```
mediaTrackers[sessionId] = null
```

but rather 

```
mediaTrackers.remove(sessionId)
```